nilearn/_utils/numpy_conversions.py

```diff
@@ -123,10 +99,7 @@ def as_ndarray(arr, copy=False, dtype=None, order="K"):
             ret = ret.T.copy().T if ret.flags["F_CONTIGUOUS"] else ret.copy()
 
```

Comment:
I do not know why this line is necessary, couldn't we treat `np.memmap` and `np.ndarray` the same way ?

Reply:
A priori I don't think we need to make a difference. But I have limited experience with mmap manipulation.

Reply:
I'll check if it makes difference in terms of efficiency.

Reply:
thanks!

Reply:
This is necessary for performance issues.

`ret.T.copy().T` takes half the time compared to `ret.copy()` when ret array is fortran style. 

However `np.array(ret, copy=True)` takes approximately the same time as `ret.T.copy().T` may be less. 

I guess this was necessary for old numpy versions. I tested for 1.22 and last numpy version and there is no need for this line any more.